modules/ROOT/pages/try-scope-concept.adoc

```diff
@@ -97,14 +100,15 @@ Suppose now we add an Error Handler at flow level:
 </flow>
 ----
 
-We analyze this new scenario:
+The behavior in this scenario is:
+
+* Ignore (`INDIFFERENT`): The transaction is _continued_. This time, the commit of the `jms:publish` message depends on the flow. Because the error is now handled by an `on-error-continue` error handler, the transaction is committed. The message read from the `jms:listener` source is consumed, and the message processed by the `jms:publish` is actually sent.
```

Comment:
DITTO edits made to the previous set of described behaviors, with the following copyedits:

"The transaction is _continued_. This time, the commit of the `jms:publish` message depends on the flow. Because the error...."
  >  "The transaction continues. Because the error ...."

"...is actually sent."  >  "is sent."